app/models/location_polygon.rb

```diff
@@ -0,0 +1,6 @@
+class LocationPolygon < ApplicationRecord
+    scope :cities, -> { where(location_type: 'cities') }
```

Comment:
4 space indentation, we use only 2 in ruby

Reply:
Dunno what happened there. Apologies!

Reply:
Seems like vscode lost my defaults as I had it set to 2